java/src/main/kotlin/io/spine/validation/java/SimpleRuleGenerator.kt

```diff
@@ -106,6 +106,9 @@ internal open class SimpleRuleGenerator(ctx: GenerationContext) : CodeGenerator(
     }
 
     override fun condition(): Expression {
+        checkNotNull(otherValue) {
+            "Expected the rule to specify `simple.other_value`, but was $rule"
```

Comment:
Please have a period at the end of the sentence.

Reply:
Since `rule` is a message and print on several lines, I'll omit the period at the end. Most often, when I see such punctuation in a log statement, I have to take a moment to realize that this is still a part of a sentence.

However, I've added a colon symbol before `rule` to separate the human-readable sentence from the rest of the data.